### DISCUSSION TITLE
Configure socks5 proxy without system change

### DISCUSSION THREAD

**Discussion by srhuston** [iterate-ch/cyberduck#14929] [Q&A]
I'm looking to use CyberDuck as an S3 client for testing and browsing things, however I have one issue that I'm not able to resolve.  In order to connect to the remote bucket, I have to proxy through a specific network.  The documentation says I can configure a SOCKS5 proxy with the system settings, however I cannot (nor would I want to) do it that way as that would direct all (or most) traffic through that proxy and I only need specific connections to go that way.  _EDIT: failed to mention this is on macOS_

For more detail, I make SSH connections to a couple different networks with the -D option and a port number, utilizing ControlMaster and ControlPersist multiplexed connections.  I then spawn individual Chrome instances using different profile paths and with options like `--proxy-server="socks5://127.0.0.1:$PROXY_PORT" --host-resolver-rules="MAP * 0.0.0.0 , EXCLUDE localhost"` so that specific Chrome instance will have its traffic proxied to the remote end.  I would like to have a way I can set a specific proxy setting in CyberDuck directly, or even via commandline options, rather than having to set it system-wide and create a list of exclusions that is basically "everything but this one host or network", which I'm not sure would even be possible.

**Comment by dkocher** [comment]
We do not currently allow to override system proxy settings [^1]. 

[^1]: #5815
